_dashboards/management/advanced-settings.md

```diff
@@ -6,18 +6,151 @@ nav_order: 40
 
 {::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/alert-icon.png" class="inline-icon" alt="alert icon"/>{:/} **Note**<br>Certain settings require you to modify [the `opensearch_dashboards.yml` file](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/config/opensearch_dashboards.yml) and restart OpenSearch Dashboards.
 {: .note}
 
 ## Required permissions
 
 To modify settings, you must have permission to make changes. See [Multi-tenancy configuration](https://opensearch.org/docs/latest/security/multi-tenancy/multi-tenancy-config/#give-roles-access-to-tenants) for guidance about assigning role access to tenants.
+
+## Advanced settings descriptions
+
+The following tables describe the core advanced settings.
+
+### General settings
+
+The following table describes the **General** settings:
+
+Setting | Description
+:--- | :---
+`csv:quoteValues`  | Defines whether to Enclose values containing special characters or multiline values with a double quotation mark `"`. Default is `On`.  |
+`csv:separator`  | Defines whether to use a specific character or string to delimit exported values. Default is `,`.  |
+`dateFormat`  | Defines the format for displaying dates. Default is `MMM D, YYYY @ HH:mm:ss.SSS`.  |
+`dateFormat:dow`  | Defines the day on which to start the week. Default is `Sunday`.  |
+`dateFormat:scaled`  | Defines the format for timestamps. The timestamps format changes depending on the time between measurements (hour, minutes, seconds, and milliseconds). The keys are time periods in [ISO8601](https://www.iso.org/iso-8601-date-and-time-format.html) format: `YYYY-MM-DD`.   |
+`dateFormat:tz`  | Defines the time zone for OpenSearch Dashboards. Default is the time zone detected by your browser.  |
+`dateNanosFormat`  | Defines the format for representing dates with nanoseconds. Default is `MMM D, YYYY @ HH:mm:ss.SSSSSSSSS`.  |
+`defaultIndex`  | Defines the default index for all indexes in an OpenSearch cluster. Default is `null`.  |
+`defaultRoute`  | Defines the gateway point. Use this setting to change the landing page for OpenSearch Dashboards. The setting must be a relative URL. Default is `/app/home`. |
+`fields:popularLimit` | Defines the number of N fields to be displayed. Default is `10`.  |
+`filterEditor:suggestValues` | Defines whether the filter editor suggests field values. Default is `Off`.  |
+`filters:pinnedByDefault`  | Defines whether filters are automatically pinned. Default is `Off.`  |
```

Comment:
We can add `each filter can be set to be pinned by clicking the filter and selecting 'Pin across all apps'`?

Reply:
@abbyhu2000 Change made. Please review and make any necessary changes.